### PR TITLE
rewrite the rewrite of policy; edit the privacy request guide and light editing of query execution

### PR DESCRIPTION
**Pull request by dougfulton**
rewrite the rewrite of policy; edit the privacy request guide and light editing of query execution

### CODE REVIEW

File: docs/fidesops/docs/guides/policies.md [hunk 1]
```diff
@@ -107,27 +101,26 @@ PUT /api/v1/policy/{policy_key}/rule
...
+    "data_category": "user.provided.identifiable.contact.email",
+  }
+]
+```
 
-### A Note About Erasing Data
+## A Note About Masking
 
-When a Policy Rule erases data, it erases the _entire_ branch given by the Target. For example, if we created an `erasure` rule with a Target of `user.provided.identifiable.contact`, _all_ of the information within the `contact` node -- including `user.provided.identifiable.contact.email` -- would be erased.
+When a Policy Rule masks data, it masks the _entire_ branch given by the Target. For example, if we created an `erasure` rule with a Target of `user.provided.identifiable.contact`, _all_ of the information within the `contact` node -- including `user.provided.identifiable.contact.email` -- would be masked.
```
Comment (seanpreston) [inline docs/fidesops/docs/guides/policies.md:121]: Is it confusing to use `obfuscate` above, and `mask` here? Should we stick to one?

File: docs/fidesops/docs/guides/privacy_requests.md [hunk 2]
```diff
@@ -3,27 +3,27 @@
...
 - How can I execute a Privacy Request?
 - How do I monitor Privacy Requests as they execute?
 - How can I integrate the Privacy Request flow into my existing support tools?
 
 Take me directly to [API docs](/api#operations-Privacy_Requests-get_request_status_api_v1_privacy_request_get).
 
 ## What is a Privacy Request?
-A Privacy Request in its simplest form describes a request by a user, to process data pertaining to their identity. Privacy Requests are currently supported in two forms, `access` and `erasure`. For more information on action types, please see [How-To: Configure Request Policies](policies.md#rule-attributes).
 
+A Privacy Request  represents a request to perform an action on a user's identity data. The Request object itself identifies the user by email address, phone number, social security number, or other identifiable information. The data that will be affected and how it's affected is described in a Policy object that's associated with the Request.
```
Comment (seanpreston) [inline docs/fidesops/docs/guides/privacy_requests.md:14]: There's an erroneous double space on this line

File: docs/fidesops/docs/guides/privacy_requests.md [hunk 3]
```diff
@@ -37,12 +37,16 @@ Privacy Requests can be executed by submitting them to Fidesops via the Privacy
 ]
 ```
 
-#### Note:
+* `external_id` is an optional  identifier of your own invention that lets you track the Privacy Request. See [How-To: Report on Privacy Requests](reporting.md) for more information.
```
Comment (seanpreston) [inline docs/fidesops/docs/guides/privacy_requests.md:40]: There is an erroneous double space on this line: `optional  identifier`

File: docs/fidesops/docs/guides/query_execution.md [hunk 4]
```diff
@@ -2,17 +2,17 @@
...
+Fidesops uses your Datasets to generate a _graph_ of the resources. Based on the identity data you provide, Fidesops then generates a specific _traversal_, which is the order of steps that will be taken to fulfill a specific request. 
 
-The graph supports both directed and non-directed edges using the optional `direction` parameter on the relation (non-directional edges may be traversed in either direction). You can always preview the queries that will be generated or manually control the order of operations by making relations explicitly directional and with the `after` dataset and collection parameters. 
+The graph supports both directed and non-directed edges using the optional `direction` parameter on the relation (non-directional edges may be traversed in either direction). You can preview the queries that will be generated or manually control the order of operations by making relations explicitly directional and with the  and Collection parameters. If you specify a Collection that can't be reached, Fidesops generates an error.
```
Comment (seanpreston) [inline docs/fidesops/docs/guides/query_execution.md:7]: There's an erroneous double space on this line between `the  and Collection parameters.`

**Review by iamkelllly — APPROVED** [review]
updated and resolved merge conflicts

**Pull request merged**